templates/download/server/choose.html

```diff
@@ -7,14 +7,17 @@
 {% block content %}
 
 <section class="p-strip--suru-topped">
+  <div class="row">
+    <p class="p-muted-heading">Get Ubuntu server</p>
+    <h1 class="js-download-option">Option 3: Manual install</h1>
+  </div>
   <div class="row u-equal-height">
     <div class="col-6">
-      <p class="p-muted-heading">Get Ubuntu server</p>
-      <h1 class="js-download-option">Option 3: Manual install</h1>
+      <h2>Ubuntu Server {{ releases.lts.full_version }}</h2>
       <p class="p-heading--four">Download and install Ubuntu Server {{ releases.lts.short_version }} LTS using a USB stick or a DVD burner</p>
       <p>The long-term support version of Ubuntu Server, including the {{ releases.openstack_lts.slug }} release of OpenStack and support guaranteed until {{ releases.lts.eol }}.</p>
       <p>
-        <a href="https://wiki.ubuntu.com/{{ releases.lts.slug }}/ReleaseNotes" class="p-link--external">Ubuntu {{ releases.lts.short_version }} LTS release notes</a>
+        <a href="https://wiki.ubuntu.com/{{ releases.lts.slug }}/ReleaseNotes" class="p-link--external">Ubuntu Server {{ releases.lts.short_version }} LTS release notes</a>
```

Comment:
Copy doc says this should be "Read Ubuntu 20.04 LTS release notes"

Reply:
I will fix, technically it is both server and desktop together